src/test/java/resources/features/new_user_creation.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Smoke Test
+
+
+  Scenario: A new user flow that includes posting, commenting and album creation and deletion
```

Comment:
What does new user flow means, you should define real business and functional scenario name. Also Description about what kind of app from business and technical aspect it is and what do you check with this should be added. Something like: "This app is Free fake API used for testing and prototyping. It is designed as representation of social network...(add some description). This tests verifies..... What is avaiable with these endpoints for example user is able to create, update and delete something."
Also Scenario name should start for example with Verify something...